spec/features/applications/show_spec.rb

```diff
@@ -150,4 +151,82 @@
       end
     end
   end
+
+  describe 'Submit an Application & No Pets on an Application' do
+    describe "when one or more pets have been added to an application and its status is 'In Progress' " do
+      it 'if condition is true: has a section to submit my application, which has a test box user to add why they are a worthy candidate' do
+        visit "/applications/#{@app4.id}"
+    
+        expect(@app4.pets.present?).to eq false
+        expect(@app4.status).to eq('In Progress')
+        expect(page).to_not have_field(:bid_reason)
+        expect(page).to_not have_button('Submit Application')
+
+        visit "/applications/#{@app2.id}"
+        
+        expect(@app2.pets.present?).to eq true
+        expect(@app2.status).to eq('Pending')
+        expect(page).to_not have_field(:bid_reason)
+        expect(page).to_not have_button('Submit Application')
+
+        visit "/applications/#{@app3.id}"
+      
+        expect(@app3.pets.present?).to eq true
+        expect(@app3.status).to eq('In Progress')
+        expect(page).to have_field(:bid_reason)
+        expect(page).to have_button('Submit Application')
+      end
+
+      describe 'when user completes the worthy candidate and clicks the submit button' do 
+        before(:each) do
+          visit "/applications/#{@app3.id}"
+    
+          fill_in(:bid_reason, with: "I aint got a friend for days and love this dog A LOT, LOT")
+        end
+        
+        it 'returns user to the application show page after they click submit' do
+          
+          click_button('Submit Application')
+
+          expect(current_path).to_not eq("/pets")
+          expect(current_path).to eq("/applications/#{@app3.id}")
+        end
+
+        it 'displays the worthy candidate reason on the show page once application is submitted' do          
+          click_button('Submit Application')
+          
+          expect(page).to have_content("I aint got a friend for days and love this dog A LOT, LOT")
+          expect(page).to have_content('Pending')
+        end
+        
+        it "updates the application status 'Pending' once an application is submitted" do
+          within("#show_data") do
+            expect(page).to_not have_content('Pending')  
+            expect(page).to have_content('In Progress')
+          end
+        
+          click_button('Submit Application')
+
+          expect(current_path).to eq("/applications/#{@app3.id}")
+          
+          within("#show_data") do
+          expect(page).to_not have_content('In Progress')
+            expect(page).to have_content('Pending')  
+          end
+        end 
+
+        it "updates the application status 'Pending' once an application is submitted" do
```

Comment:
Copied 'it' description from test above. Should read something like 'Add a Pet section is no longer visible when application is pending.'